test/tools/ossfuzz/README.md

```diff
@@ -9,19 +9,18 @@ To help oss-fuzz do this, we (as project maintainers) need to provide the follow
 - test harnesses: C/C++ tests that define the `LLVMFuzzerTestOneInput` API. This determines what is to be fuzz tested.
 - build infrastructure: (c)make targets per fuzzing binary. Fuzzing requires coverage and memory instrumentation of the code to be fuzzed.
 
-Test harnesses and cmake script is provided in this directory. Apart from this, fuzzing configuration may be needed to make customize the fuzzing experience per fuzzing binary. To do this, we maintain a file with the `.options` extension. This file defines the fuzzing configuration as follows:
-  - `close_fd_mask`: When equal to `3` this pipes console output (stdout/stderr) generated by the test harness to `/dev/null`
-  - `dict`: Name of the dictionary file to be used for fuzz testing. A [dictionary file][2] contains a newline-seperated-block-quote-enclosed set of strings
-
 ## What is libFuzzingEngine.a?
 
-`libFuzzingEngine.a` is a oss-fuzz-related dependency. It is present in the Dockerized environment in which Solidity's oss-fuzz code will be built.
+`libFuzzingEngine.a` is an oss-fuzz-related dependency. It is present in the Dockerized environment in which Solidity's oss-fuzz code will be built.
 
 ## Is this directory relevant for routine Solidity CI builds?
 
-No. This is the reason why the `add_subdirectory(ossfuzz)` cmake directive is nested under the `if (DEFINED ENV{OSSFUZZ})` predicate.
+No. This is the reason why the `add_subdirectory(ossfuzz)` cmake directive is nested under the `if (OSSFUZZ)` predicate. `OSSFUZZ` is a new solidity-wide cmake option that can be enabled like so:
```

Comment:
Please don't use `new` - it would mean that we have to change this file in 3 months.

Reply:
done